### DISCUSSION TITLE
Laptop vs Desktop

### DISCUSSION THREAD

**Discussion by ItalyPaleAle** [plausible/analytics#379] [Anything Else]
## Prerequisites
- [X] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report
The UI considers all visitors with a screen wider than 1440px to be "desktops", and the smaller ones are "laptop". This is a bit of an arbitrary number that doesn't keep into account modern laptops that have high-res screens.

**Expected behavior**
"Desktops" and "laptops" cannot reliably be distinguished by the screen resolution, so they should be grouped together. The reason is that when separate, they cause a "distortion" on the analytics, that might make it look like mobile gets a larger share.

**Comment by ukutaht** [comment]
That's fair, I agree we should change it.

At the moment it's working as intended so I will remove the bug label.

**Reply by justrealmilk** [reply]
Looked through the docs to find the distinction but couldn't. Thankfully I found this. 

I think mobile, tablet, small desktop display, large desktop display would be better labels

**Comment by metmarkosaric** [comment]
**Update:** This is now live! From today, the devices are categorized into desktop, mobile or tablet. This is derived from the User-Agent HTTP header. The full User-Agent is discarded.
